library/src/main/java/com/google/android/material/motion/runtime/TargetScope.java

```diff
@@ -118,6 +134,23 @@ private int getDetailedState() {
     return state;
   }
 
+  private void removeNamedPlan(PlanInfo planInfo, Performer performer) {
```

Comment:
this ties into a previous comment - i would rather not see `PlanInfo` as a param type.